packages/application-shell/src/components/project-switcher/messages.ts

```diff
@@ -1,10 +1,10 @@
 import { defineMessages } from 'react-intl';
 
 export default defineMessages({
-  projects: {
-    id: 'ProjectSwitcher.projects',
+  projectsLabel: {
```

Comment:
`projects` key message was not used so I decided to remove it and create a new one more specific to the context where it's used.